terraform/cluster.tf

```diff
@@ -41,6 +29,26 @@ resource "aws_instance" "app_server" {
   ]
 }
 
+resource "aws_db_instance" "db" {
+  identifier           = "${var.cluster_name}-db-${count.index}"
+  allocated_storage    = 100
+  storage_type         = "io1"
+  iops                 = 1000
+  engine               = var.db_instance_engine
+  engine_version       = var.db_engine_version[var.db_instance_engine]
+  instance_class       = "db.r4.large"
```

Comment:
can we make `instance_class` a variable?

Reply:
done